app/views/shared/layout/nav/nav.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import styles from './nav.scss';
+import ba_btn_image from "app/assets/bastrap3/ba-btn.png";
+import { Link } from 'react-router'
+
+export default class Nav extends Component {
+    
+    static get propTypes() {
+        return {
+            title: PropTypes.string,
+            links: PropTypes.array
+        };
+    }
+
+    static defaultProps = {
+        links: [
+            {
+                refTo: "/",
+                displayName: "Inicio",
+                onlyActiveOnIndex: true,
+            }
+        ],
+        title: "GCBA"
+    }
+
+    render() {
+        return (
+            <nav className="navbar navbar-default" role="navigation">
+                <div className="container">
+                  <div className="row">
+                      <div className="navbar-header">
+                          <a className="navbar-brand" href="#" title={"this.props.title"}>{this.props.title}</a>
+                      </div>
+                      <div className="collapse navbar-collapse" id="main-nav">
+                          <ul className="nav navbar-nav navbar-right">
+                              {this.props.links.map(function(link,index){
```

Comment:
extraer el map dentro de una funcion que diga, **renderLinks() {}** y dentro del html llamar directamente la funcion **{this.renderLinks()}** esto nos permite tener mas limpio el html y encaptular las funcionalidades en funciones, haciendo mucho mas facil de mantener y leer el codigo